components/ai_chat/resources/page/components/feature_button_menu/index.tsx

```diff
@@ -95,12 +95,23 @@ export default function FeatureMenu() {
         </div>
       </leo-menu-item>
 
+      {!context.isPremiumUser &&
       <leo-menu-item onClick={context.goPremium}>
```

Comment:
Not sure what the style guide says but maybe this should be indented (but also, maybe not :laughing: )

Reply:
I prefer to not indent when there is no new html element hierarchy level, but I agree it's not perfect. This is what I've been doing for years though 🤷